cla_public/templates/macros/element.html

```diff
@@ -44,8 +44,9 @@ <h3 class="alert-header" id="alert-heading-{{type}}">{{ _(title) }}</h3>
 {%  macro satisfaction_survey_prompt() %}
   {% call alert(title=_('What do you think about this service?')) %}
     <p id="aria-satisfaction-survey">
-      {% trans %}Your feedback will improve this service.
-      Please <a href="https://www.gov.uk/done/check-if-civil-legal-advice-can-help-you" aria-labelledby="aria-satisfaction-survey">send us your thoughts</a>.{% endtrans %}
+      {% trans -%}
+        Your feedback will improve this service. Please <a class="govuk-link" href="https://www.gov.uk/done/check-if-civil-legal-advice-can-help-you">send us your thoughts</a>.
```

Comment:
Do we need translate the link url. If what do you think of doing something like:

```
{% set feed_back_link = '<a class="govuk-link" href="https://www.gov.uk/done/check-if-civil-legal-advice-can-help-you">' %}
{% trans -%}Your feedback will improve this service. Please {{ feed_back_link }} send us your thoughts{%- endtrans %}</a>.
```

Reply:
At the moment I think you are right.  But what if the link to the feedback page changes if in Welsh - we need to link to a Welsh feedback page with a different URL.